pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -21,8 +21,14 @@ import (
 )
 
 const (
-	AlertmanagersKind   = "Alertmanager"
-	AlertmanagerName    = "alertmanagers"
+	// Deprecated: Use github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring
+	// TODO: Remove in v0.75
```

Comment:
```suggestion
	// TODO: Remove in v0.75
	// Deprecated: Use github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring instead of github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1. This constant will be removed in v0.75
```

I would just make the deprecation message more clear (see screenshot above)

Reply:
The main problem I see in the message is this "TODO: Remove in v0.75" is displayed downstream. If we change the order, the TODO shouldn't be displayed anymore